.stylelintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+      "indentation": 4,
+      "no-empty-source": null
```

Comment:
Which files violate this?

Reply:
Some of the CSS files that MarkBind generates upon initialisation are empty (e.g. `docs/_markbind/layouts/default/styles.css`).

Reply:
Might want to document this case in the commit message so that future developers know why this rule `no-empty-source` was added.